api/internal/target/kusttarget.go

```diff
@@ -251,7 +252,12 @@ func (kt *KustTarget) configureExternalGenerators() ([]resmap.Generator, error)
 	if err != nil {
 		return nil, err
 	}
-	return kt.pLdr.LoadGenerators(kt.ldr, kt.validator, ra.ResMap())
+	m := ra.ResMap()
+	err = m.AnnotateAll(kioutil.WorkingDirAnnotation, kt.ldr.Root())
```

Comment:
Don't we also need to do this in the equivalent places for transformers and validators?

Reply:
done